quantum-template.tex

```diff
@@ -25,6 +25,8 @@
 \affiliation{ICFO-Institut de Ciencies Fotoniques, The Barcelona Institute of Science and Technology, 08860 Castelldefels (Barcelona), Spain}
 \author{Marcus Huber}
 \affiliation{Institute for Quantum Optics \& Quantum Information (IQOQI), Austrian Academy of Sciences, Boltzmanngasse 3, Vienna A-1090, Austria}
+\author{Christopher Granade}
+\author{Johannes J. Meyer}
 
```

Comment:
```suggestion
\orcid{0000-0003-1533-8015}
```

I'm a big fan of the concept behind ORCiD, so listing it for all authors would be nice to inspire more people, maybe?

Reply:
And thanks for mentioning me! Would not have been necessary.